script/DeploySpark.s.sol

```diff
@@ -125,69 +169,77 @@ contract DeploySpark is Script {
-        ScriptTools.exportContract(instanceId, "poolAddressesProviderRegistry", address(registry));
-        ScriptTools.exportContract(instanceId, "poolAddressesProvider", address(poolAddressesProvider));
-        ScriptTools.exportContract(instanceId, "protocolDataProvider", address(protocolDataProvider));
-        ScriptTools.exportContract(instanceId, "poolConfigurator", address(poolConfigurator));
-        ScriptTools.exportContract(instanceId, "poolConfiguratorImpl", address(_poolConfigurator));
-        ScriptTools.exportContract(instanceId, "pool", address(pool));
-        ScriptTools.exportContract(instanceId, "poolImpl", address(_pool));
-        ScriptTools.exportContract(instanceId, "aclManager", address(aclManager));
-        ScriptTools.exportContract(instanceId, "aaveOracle", address(aaveOracle));
-        ScriptTools.exportContract(instanceId, "aTokenImpl", address(aTokenImpl));
-        ScriptTools.exportContract(instanceId, "stableDebtTokenImpl", address(stableDebtTokenImpl));
-        ScriptTools.exportContract(instanceId, "variableDebtTokenImpl", address(variableDebtTokenImpl));
-        ScriptTools.exportContract(instanceId, "treasury", address(treasury));
-        ScriptTools.exportContract(instanceId, "treasuryImpl", address(treasuryImpl));
-        ScriptTools.exportContract(instanceId, "treasuryController", address(treasuryController));
-        ScriptTools.exportContract(instanceId, "incentives", address(incentives));
-        ScriptTools.exportContract(instanceId, "incentivesImpl", address(rewardsController));
+        ScriptTools.exportContract(instanceId, "aTokenImpl",      address(aTokenImpl));
+        ScriptTools.exportContract(instanceId, "aaveOracle",      address(aaveOracle));
+        ScriptTools.exportContract(instanceId, "aclManager",      address(aclManager));
+        ScriptTools.exportContract(instanceId, "admin",           address(admin));
+        ScriptTools.exportContract(instanceId, "deployer",        address(deployer));
         ScriptTools.exportContract(instanceId, "emissionManager", address(emissionManager));
-        ScriptTools.exportContract(instanceId, "uiPoolDataProvider", address(uiPoolDataProvider));
+        ScriptTools.exportContract(instanceId, "incentives",      address(incentives));
+        ScriptTools.exportContract(instanceId, "incentivesImpl",  address(rewardsController));
+        ScriptTools.exportContract(instanceId, "pool",            address(pool));
+
+        ScriptTools.exportContract(instanceId, "poolAddressesProvider",         address(poolAddressesProvider));
+        ScriptTools.exportContract(instanceId, "poolAddressesProviderRegistry", address(registry));
+
+        ScriptTools.exportContract(instanceId, "poolConfigurator",        address(poolConfigurator));
+        ScriptTools.exportContract(instanceId, "poolConfiguratorImpl",    address(poolConfigurator));
+        ScriptTools.exportContract(instanceId, "poolImpl",                address(pool));
```

Comment:
This is also incorrectly referencing the pool instance instead of the implementation.